polytope/polytope.py

```diff
@@ -442,6 +468,134 @@ def text(self, txt, ax=None, color='black'):
         _plot_text(self, txt, ax, color)
 
 
+def _translate(polyreg, d):
+    """Translate a polyreg by a vector in place. Does not return a copy.
```

Comment:
I am happy with both "in-place" and "in place". Given there is no reliable distinction that we have found, we can wait for empirical feedback from users.